endaq/ide/files.py

```diff
@@ -20,7 +25,49 @@
 from .info import parse_time
 from .util import validate
 
-__all__ = ['get_doc', 'extract_time']
+__all__ = ["get_doc", "extract_time"]
+
+# ============================================================================
+#
+# ============================================================================
+
+
+def normalized_path(
+    filepath: str, mode: typing.Literal[None, "local", "url"] = None
+) -> Union[
+    Tuple[str, typing.Literal["local"]], Tuple[ParseResult, typing.Literal["url"]]
```

Comment:
I'm really gonna nitpick here, sorry.  Can we separate longer `Union` statements onto separate lines, such that each option for the union is separate from the rest?

Reply:
you meant like ab156e1434c1a2bd0b73cbf2f356d657fc2d6b36, right?